tests/JsonApiErrorMiddlewareTest.php

```diff
@@ -34,6 +42,8 @@ public function dummyTest()
         $this->assertTrue(true);
     }
 
+    // TODO NOW: Tried that, but doesn't work because exception without message, status and code always has empty body
```

Comment:
Like stated above, in my opinion even an empty throwable should have some information (at least a very basic one) if possible.

Reply:
If we don't have any information we decided to set a default http status code and a default error message.